### PR TITLE
update cmake lists to allow wgs84_utils to be used in other packages

### PR DESCRIPTION
**Pull request by msmcconnell**
# PR Details
Updates sensor fusion CMakelists.txt file to export the wgs84_utils library so that other ros packages can use it. Additionally copies over the ecef_to_geodesic function from carmajava geometry package

## Related Issue
#256 

## Types of changes
- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](Contributing.md) 
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: sensor_fusion/src/wgs84_utils/wgs84_utils.cpp [hunk 1]
```diff
@@ -155,6 +141,43 @@ tf2::Vector3 wgs84_utils::geodesic_to_ecef(const wgs84_utils::wgs84_coordinate &
     return point_in_ned;
 }
 
+  /**
+   * Converts a given 3d cartesian point in the ECEF frame into a WSG-84 geodesic location
+   * The provided point should be defined relative to a frame which has a transform with the ECEF
+   * @param point The cartesian point to be converted
+   * @return The calculated WSG-84 geodesic location
+   */
+  wgs84_utils::wgs84_coordinate wgs84_utils::ecef_to_geodesic(const tf2::Vector3& point) {
```
Comment (qswawrq) [inline sensor_fusion/src/wgs84_utils/wgs84_utils.cpp]: Can you add a simple unit test for this to make sure the math is correct?

Comment (qswawrq) [reply sensor_fusion/src/wgs84_utils/wgs84_utils.cpp]: Just realize you said it is from geometry package. So I assume it should be correct. Maybe just copy the unit test over as well?

Comment (msmcconnell) [reply sensor_fusion/src/wgs84_utils/wgs84_utils.cpp]: Yeah good idea

Comment (msmcconnell) [reply sensor_fusion/src/wgs84_utils/wgs84_utils.cpp]: Removed for now as function is unused per our dicsussion

**Pull request merged**